### DISCUSSION TITLE
Is there a way to get a more verbose output of the error failing my build?

### DISCUSSION THREAD

**Discussion by jamesdavieswales** [parcel-bundler/parcel#7523] [Q&A] [answered]
I am currently looking into switching our companies bundle tool from webpack to parcel. I have ran into a couple of issues, however I have been able to unblock myself for the most part. 

I believe I'm on my final issue/s. I am running my build with `parcel build src/assets/index.html --log-level verbose` and I'm having the error output 

`🚨 Build failed.

@parcel/optimizer-terser: Missing initializer in const declaration
`

This isn't telling me where specifically the missing dec'd const lives (bare in mind, this is a very large application) the logs above this build error are a few warnings - it seems to fail on the optimization steps. Is it possible to get the logging to tell me exactly where it failed?

**Comment by mischnic** [comment, marked as answer]
It tries to use the sourcemap to trace back the origin of that error, so that might be a problem if you have a very large app (or if the sourcemaps are invalid for some reason). Using `--no-source-map` to make it print the error in the actual bundle output should work (and there should be a codeframe after that error message).

Otherwise, you could do `--no-optimize` and try to run the app or minify the bundle(s) manually to find the invalid one.

(And this is definitely a bug in Parcel because the input to Terser should always be valid JS)

**Reply by jamesdavieswales** [reply]
Thanks, source maps gave me a seg fault line so not too much to go on. The no opt has unblocked me as I've gone digging into the compiled source to find it - Thank you!
